### ISSUE TITLE
Some loader's `asyncLoad` argument type is incorrect

### ISSUE THREAD

**Issue by shun-shobon** [three-types/three-ts-types#309]
-   `three` version: `0.148.0`
-   `node` version: `18.12.1`
-   `yarn` version: `3.3.1`

### Problem description:

The `url` type of `asyncLoad` in `CubeTextureLoader` differs from `load`.
Originally `string[]` in `load` is correct, but since `loadAsync`'s is `string`, it is impossible to load textures correctly when using `loadAsync`.

https://github.com/three-types/three-ts-types/blob/master/types/three/src/loaders/CubeTextureLoader.d.ts#L15

I have not tested it, but it seems that the `HDRCubeTextureLoader` type definition is also incorrect.

https://github.com/three-types/three-ts-types/blob/master/types/three/examples/jsm/loaders/HDRCubeTextureLoader.d.ts

### Relevant code:

```ts
import * as THREE from "three"
const loader = new THREE.CubeTextureLoader();

const urls = ["right.png", "left.png", "top.png", "bottom.png", "front.png", "back.png"];

// it's correct.
loader.load(urls, (texture) => { /* ... */ });

// it's also correct, but will result in a type error.
const texture = await loader.asyncLoad(urls);
```

### Suggested solution:

I propose to remove the `loadAsync` definition from the base `Loader.d.ts` and define it in each loader.
It is necessary to define `loadAsync` in all loaders that inherit `Loader`, but fortunately, most of them have it defined, so we only need to define the missing part.

**Comment by 0b5vr** [comment]
Agreed. Would you send us a PR?

**Comment by shun-shobon** [comment]
Sure. I will create a PR.

**Comment by shun-shobon** [comment]
I created #311.

**Comment by 0b5vr** [comment]
resolved with #311
